enhancements/installer/component-selection.md

```diff
@@ -92,235 +97,185 @@ an install experience that disables components that aren't needed for my use cas
-values in the annotation(e.g. `capability.openshift.io/name=console+monitoring`).  The resource will only be 
-included if all the capabilities on the resource are enabled.  This allows a component
-to define a `ServiceMonitor` which is part of both the `Monitoring` capability and the `Foo` capability.  If
-either of those capabilities are excluded, the `ServiceMonitor` should not be created.  (If Monitoring is excluded
-then no ServiceMonitors should be created, if only Foo is excluded, then only Foo's ServiceMonitors should be
-excluded while ServiceMonitors associated with other capabilities would not be).
+The installer will validate the pass the information through to the CVO for resource management, by setting [`spec.capabilities` in ClusterVersion](#resource-management).
 
-Specifying the same capability as an `include` and an `exclude` is an error condition.  This avoids the need 
-to define behavior for what to do when a user specifies the same capability in both, and update a status field
-indicating what choice the CVO made.  To enforce this the CVO will need to evaluate the spec values and, if
-they are valid, copy them into status.  If they are not valid, status must contain an indication of what is
-invalid and why.  This also means the CVO must be driven off the status values, not the spec values.
+#### Resource management
 
-Note: The CVO must wait until the ClusterVersion resource exists (created by the installer) before creating
-any filterable resource(a resource with a capability annotation) to ensure there is no race condition in which
-the CVO starts creating resources that should have been filtered, before it has the filter list.
+The CVO will grow new `capabilities` properties in `spec` and `status`.
+This is distinct from `overrides` because `overrides` put the cluster in an unsupported state.
+
+The `spec` property uses the same structure as [the install-config](#install-configuration):
+
+```yaml
+  spec:
+    capabilities:
+      baselineCapabilitySet: None
+      additionalEnabledCapabilities:
+      - openshift-samples
+```
 
+The CVO will calculate an effective status:
 
-Strawman of status api, this will be iterated on in api review.
 ```yaml
   status:
     capabilities:
-      inclusionDefault: [Include|Exclude]
-      exclude:
-      - capability1
-      - capability2
-      include:
-      - capability3
-      - capability4
-    conditions:
-      - condition1
-      - condition2
+      enabledCapabilities:
+      - openshift-samples
+      knownCapabilities:
+      - baremetal
+      - marketplace
+      - openshift-samples
 ```
 
-Conditions will reflect the validity/invalidity of the spec-requested configuration.
-status.capabilities will reflect the current enforced capability set.
+`knownCapabilities` can inform users of available but not currently enabled capabilities (e.g. web-console checkboxes for "do you want to enable these additional capabilities?").
```

Comment:
I do not think this is true.  `knownCapabilities` should list all known capabilities and should not exclude the currently enabled capabilities  
```
$ oc get -o json clusterversion version | jq '{spec: (.spec | {capabilities}), status: (.status | {capabilities})}'
  {
    "spec": {
      "capabilities": {
        "baselineCapabilitySet": "None",
        "additionalEnabledCapabilities": [
          "openshift-samples"
        ],
      },
    },
    "status": {
      "capabilities": {
        "enabledCapabilities": [
          "openshift-samples"
        ],
        "knownCapabilities": [
          "baremetal",
          "marketplace",
          "openshift-samples"
        ]
      }
    }
  }
```

Reply:
rephase to `can inform users of all capabilities the cluster understands regardless of whether they are enabled or not` should resolve this.

Reply:
I've pushed 5f94ef77 -> 523c93d5 with:

```diff
-`knownCapabilities` can inform users of available but not currently enabled capabilities (e.g. web-console checkboxes for "do you want to enable these additional capabilities?").
-
+`knownCapabilities` includes all known capabilities, regardless of whether they are enabled or not.
+Removing `enabledCapabilities` from `knownCapabilities` will give the set of available but not currently enabled capabilities (e.g. web-console checkboxes for "do you want to enable these additional capabilities?").
```